packages/expo-updates/src/Updates.types.ts

```diff
@@ -20,21 +22,6 @@ export enum UpdateEventType {
   ERROR = 'error',
 }
 
-// @docsMissing
-// TODO(eric): move source of truth for manifest type to this module
-/**
- * @hidden
- */
-export type ClassicManifest = typeof Constants.manifest;
```

Comment:
this is technically a breaking change but is fine since it was being removed anyways in 50